src/util/arr.h

```diff
@@ -26,16 +26,20 @@
 #include <stdarg.h>
 #include <stdint.h>
 #include <assert.h>
-#include "rmalloc.h"
+#include <stdio.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
 
 /* Definition of malloc & friedns that can be overridden before including arr.h.
  * Alternatively you can include arr_rm_alloc.h, which wraps arr.h and sets the allcoation functions
  * to those of the RM_ family
  */
 #ifndef array_alloc_fn
-#define array_alloc_fn rm_malloc
-#define array_realloc_fn rm_realloc
-#define array_free_fn rm_free
+#define array_alloc_fn malloc
+#define array_realloc_fn realloc
+#define array_free_fn free
```

Comment:
Was this a change for debug purposes? I'd definitely prefer to use the rm_ family for arrays.

Reply:
👍